lib/common/repository/db_repository.dart

```diff
@@ -235,6 +235,7 @@ class DBRepository {
     BookmarkStatus? bookmarkStatus,
     String? sessionLevel,
     String? sessionType,
+    String? query,
```

Comment:
Since you have gone with dedicated search methods, you can shave this attribute.

Reply:
fixed